node/node_handler.go

```diff
@@ -443,14 +464,20 @@ func (node *Node) PostConsensusProcessing(
 		node.BroadcastCXReceipts(newBlock, commitSigAndBitmap)
 	} else {
 		if node.Consensus.Mode() != consensus.Listening {
+			sigs := "none"
+			if sigsInBlock, err := node.signaturesIncludedInBlock(newBlock); err == nil {
+				if len(sigsInBlock) > 0 {
+					sigs = strings.Join(sigsInBlock, ",")
+				}
+			}
 			utils.Logger().Info().
 				Uint64("blockNum", newBlock.NumberU64()).
 				Uint64("epochNum", newBlock.Epoch().Uint64()).
 				Uint64("ViewId", newBlock.Header().ViewID().Uint64()).
 				Str("blockHash", newBlock.Hash().String()).
 				Int("numTxns", len(newBlock.Transactions())).
 				Int("numStakingTxns", len(newBlock.StakingTransactions())).
-				Msg("BINGO !!! Reached Consensus")
+				Msgf("BINGO !!! Reached Consensus, signatures included: %s", sigs)
```

Comment:
Better to only show "BINGO !!! Reached Consensus; Signatures Included: %d", len(sigs)

No one really care about the signature detail. Just whether it's included or not.